src/openeo_gfmap/stac/constants.py

```diff
@@ -0,0 +1,232 @@
+"""
+Constants in the STAC collection generated after a series of batch jobs
+"""
+import pystac
+
+TITLE = "WorldCereal Level 1 cache"
+ID = "L1-CACHE"
```

Comment:
What does `L1` mean here?

Reply:
Removed, but that's something @VincentVerelst implemented